src/js/_enqueues/lib/nav-menu.js

```diff
@@ -601,6 +604,26 @@
 					titleEl.text( wp.i18n._x( '(no label)', 'missing menu item navigation label' ) ).addClass( 'no-title' );
 				}
 			} );
+
+			// Show warning when url is empty/invalid in custom menu item type.
+			menuToEdit.on( 'change input', '.edit-menu-item-url', function( e ) {
+
+				const input = $( e.currentTarget );
```

Comment:
Same as before, use var instead on const. Also instead of declaring var multiple time, make it comma separated. For Reference check code from 597 to 599 above. Also no extra empty line.

Reply:
- Resolved in https://github.com/WordPress/wordpress-develop/pull/6389/commits/24598ed85737201ee50112246c4c0b6da3458029 🚀 .

Reply:
@Pathan-Amaankhan Var code needs to be something like this:

`
var input = $( e.currentTarget ),
	url = input.val(),
	urlEl = input.closest( '.menu-item' ).find( '.field-url' );
`